common/certificate/index.ts

```diff
@@ -78,6 +84,64 @@ export async function getCertificatePDF(certificateId: string, requestor: Studen
     return pdf;
 }
 
+export async function createInstantCertificate(requester: Student, lang: Language): Promise<{ pdf: Buffer; certificate: InstantCertificate }> {
+    const matchesCount = await prisma.match.count({ where: { studentId: requester.id } });
+    const matchAppointmentsCount = await prisma.lecture.count({ where: { match: { studentId: requester.id } } });
+    const uniqueCourseParticipants = await prisma.subcourse_participants_pupil.groupBy({
```

Comment:
nit: Would it make sense to combine all of the queries into a `Promise.all` call?